### ISSUE TITLE
Web "& Design" team

### ISSUE THREAD

**Issue by carlaberkers** [ubuntudesign/webteam-practices#109]
We are actually the Web and Design team - we should be inclusive in the website (and repo!) title
#designtoo 

![screen shot 2018-11-21 at 10 24 26](https://user-images.githubusercontent.com/7945048/48835321-c661f600-ed77-11e8-8722-6e727687a63a.png)

**Comment by nottrobin** [comment]
I agree with this. A quick solution could be to change the repository name to "web-and-design-practices" and change the text in the `base.html`, `index.md` and `README.md`. But it would still sit under the "canonical-webteam" organisation.

To resolve that, I think we should look at our GitHub organisation structure in general. Given that repositories inside organisations [can now be grouped under "topic"s](https://blog.github.com/2017-01-31-introducing-topics/), the remaining reason we maintain separate organisations from CanonicalLtd is because we so far haven't been allowed general admin access to that organisation. If we could get general admin access for managers at least, I would be happy to restructure everything under there. Then this repo would become `canonicalltd/web-and-design-practices`, which I think would be ideal.

**Comment by nottrobin** [comment]
https://github.com/canonical-webteam/practices/pull/114 changed the wording in the website and content to only mention the "Web and design team".

Renaming the repository is more tricky because it will change the link of https://canonical-webteam.github.io/practices/ (the old URL [will just 404](https://help.github.com/articles/renaming-a-repository/)). Which seems a shame given  I *just* emailed the link around to everyone.

I propose we leave the name of the repository itself the same until we have decided what we're doing with the `canonical-webteam` organisation portion. Then we can move the website all at once.
